### PR TITLE
Isogram approaches

### PR DESCRIPTION
**Pull request by bobahop**
Initial commits for file structure and code. Explanatory details to follow...

### CODE REVIEW

**Comment by bobahop** [comment]
@ErikSchierboom Isogram is ready for review

File: exercises/practice/isogram/.approaches/bitfield/content.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Bit field
+
+```csharp
+public static class Isogram
+{
+    public static bool IsIsogram(string word) {
+    int letter_flags = 0;
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/bitfield/content.md:7]:
```suggestion
        int letter_flags = 0;
```

File: exercises/practice/isogram/.approaches/bitfield/content.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Bit field
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/bitfield/content.md:1]: Love this solution, as it is quite unconventional for C# programmers to write it like this (which makes it interesting)

File: exercises/practice/isogram/.approaches/bitfield/snippet.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+if (letter >= 'a' && letter <= 'z')
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/bitfield/snippet.txt:1]: Nice summary

File: exercises/practice/isogram/.approaches/introduction.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+}
+```
+
+For more information, check the [`GroupBy` approach][approach-groupby].
+
+## Other approaches
+
+Besides the aforementioned, idiomatic approaches, you could also approach the exercise as follows:
+
+## Bit field approach:
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.approaches/introduction.md:51]: For consistency:

```suggestion
### Other approach: Bit field
```

File: exercises/practice/isogram/.articles/performance/content.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (ErikSchierboom) [inline exercises/practice/isogram/.articles/performance/content.md:1]:
```suggestion
```

**Review by ErikSchierboom — APPROVED** [review]
Awesome!

**Pull request merged**